src/Mealz/MealBundle/Controller/BaseController.php

```diff
@@ -36,39 +42,31 @@ public static function getSubscribedServices(): array
      */
     public function getMealRepository()
     {
-        return $this->getDoctrine()->getRepository('MealzMealBundle:Meal');
-    }
-
-    /**
-     * @return DishRepository
-     */
-    public function getDishRepository()
```

Comment:
Is this method gone now?

Reply:
Yes, we didn't use it. I could reintroduce it and use it once in the MealAdminController. Does it make sense?

Reply:
It makes no sense to keep it if it's not used.